collectors/ebpf.plugin/ebpf.c

```diff
@@ -1175,26 +747,28 @@ int main(int argc, char **argv)
     int i;
     int end = NETDATA_EBPF_PROCESS_THREADS;
```

Comment:
what is reason to define `NETDATA_EBPF_PROCESS_THREADS` if we can know it from size of `ebpf_modules` array?

Reply:
I kept it only to define the array size.

Reply:
Yes but i would prefer to allocate based on ebpf_modulles size so you not have opportunity to forget to change size when you add plugins. But i keep it up to you

Reply:
I thought again about this case, and I removed the `define`, thank you for the suggestion!